src/views/Portal/components/ApplicationStatus/index.view.tsx

```diff
@@ -28,6 +28,9 @@ const ApplicationStatus: React.FC = () => {
         setStatusMessage("NOT STARTED")
         // setStatusMessage("CLOSED") Uncomment to Close Applications
         break
+      // case AppStatus.InProgress:
```

Comment:
I'll implement this in a separate PR along with the progress bar update once we get design's approval.